src/components/Clock.vue

```diff
@@ -79,18 +79,21 @@ export default {
       }
     },
     showTime() {
-      const date = new Date()
+      const timeZone = -4 // GTM -4:00
+
+      const d = new Date()
+      const utc = d.getTime() + (d.getTimezoneOffset() * 60000)
+      // getTime -> milliseconds since epoch (absolute time)
```

Comment:
Thanks for share the timezone match formula. This could be helpful for future time-sensitive projects.